src/data/variables.json

```diff
@@ -0,0 +1,76 @@
+{
+    "sdkx_android": {
+        "sdk_version": "10.2.0",
+        "sdk_url": "https://d3e51fp79zp4el.cloudfront.net/library/latest/sdkx/helpshift-plugin-sdkx-android.zip"
+    },
+    "sdkx_ios": {
+        "sdk_version": "10.2.0",
```

Comment:
The latest SDK X iOS version is 10.2.1. 
Will updating the version here be part of the SDK X dev docs release process? Please work with the mobile team to create a guide for SDK release dev docs changes.

Reply:
Yes. This is similar to updating the settings.json file in `quill`. I will be creating a document for how this will be updated for SDK X releases and sharing it soon.

Reply:
Okay. Please create a backlog and resolve this comment.

Reply:
Added backlog task